api/apps/v1alpha1/common_types.go

```diff
@@ -35,7 +35,8 @@ type Service struct {
 	// override the default service name
 	Name string `json:"name,omitempty"`
 	// +kubebuilder:default=8000
-	OpenAIPort int32 `json:"openaiPort"`
+	OpenAIPort         int32             `json:"openaiPort"`
+	ServiceAnnotations map[string]string `json:"serviceAnnotations,omitempty"`
```

Comment:
This should be just `Annotations` as service.annotations is self explanatory and also it stutters.

Reply:
Fixed